spec/app/create_policy_spec.rb

```diff
@@ -28,7 +28,7 @@
         'repo'          => repo.name,
         'task'          => 'some_os',
         'broker'        => broker.name,
-        'hostname'      => "host${id}.example.com",
+        'hostname'      => "host<%= id %>.example.com",
```

Comment:
Following the backward-compatibility suggestion above, this would need to be split into two test cases.